taskcluster/app_services_taskgraph/build_config.py

```diff
@@ -0,0 +1,42 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import os
+import yaml
+
+from taskgraph.util.memoize import memoize
+
+
+def get_components():
+    build_config = _read_build_config()
+    return [{
+        'version': build_config['libraryVersion'],
```

Comment:
This was for convenience, so that the component version is with the rest of the component data.
I cache it in global state and have consumers call `build_config.get_version()`, but then it would be global :man_shrugging:. I'm indifferent, let me know what you think

Reply:
Let's keep it this way for now, and revisit later 🙂